MCP_DAC.cpp

```diff
@@ -70,6 +70,19 @@ void MCP_DAC::begin(uint8_t select)
       mySPI->end();
       mySPI->begin(18, 19, 23, select);   // CLK=18 MISO=19 MOSI=23
     }
+  
+    #elif defined(ARDUINO_ARCH_RP2040)
+    
+    if (_useSPI == false &&  _useSPI1 == true) {
```

Comment:
prefer extra brackets so evaluation is 100% clear.
 if ((_useSPI == false) && (_useSPI1 == true))